frontend/public/models/index.ts

```diff
@@ -984,3 +984,17 @@ export const VolumeSnapshotContentModel: K8sKind = {
   labelPlural: 'Volume Snapshot Contents',
   crd: true,
 };
+
+export const ClusterServiceVersionModel: K8sKind = {
```

Comment:
Can this be removed since you dropped the other commit?

Reply:
All set!